docs/src/components/DocumentMetadata/index.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useEffect, useState } from 'react'
+import styles from './styles.module.css'
+import metadatas from '@site/static/metadatas.json'
+
+interface Props { }
+
+interface Metadata {
+  lastUpdatedAt: string
+  relativeTimeSince: string
+  commitHash: string
+}
+
+const Utils = {
+  getRelativeMillisTimeSince: (lastUpdatedAt: string, lastTranslatedAt: string) => {
+    const docLastUpdatedAt = new Date(lastUpdatedAt).getTime()
+    const docLastTranslatedAt = new Date(lastTranslatedAt).getTime()
+    const relativeTimeSince = docLastTranslatedAt - docLastUpdatedAt
+    return relativeTimeSince
+  }
+  , getTimeObject: (relativeTime: number) => {
+    const relativeUnits = [
+      { unit: 'seconds', divisor: 1000 },
+      { unit: 'minutes', divisor: 1000 * 60 },
+      { unit: 'hours', divisor: 1000 * 60 * 60 },
+      { unit: 'days', divisor: 1000 * 60 * 60 * 24 }
+    ]
+
+    const relativeTimes = relativeUnits
+      .map(({ unit, divisor }) => {
+        const value = Math.floor(relativeTime / divisor)
+        return { value, unit }
+      })
+
+    return relativeTimes.reduce((acc, obj) => {
+      return (
+        // discard when relative time of translated >= default language
+        obj.value <= 0 &&
+        // we are interested to find the maximum negative relative time in the list
+        obj.value >= acc.value
+      ) ? obj : acc
+    })
+  }
+}
```

Comment:
Using a library like `momentjs` this whole code here would be quite simple. For example
```
`${moment(translationChangedAt).from(sourceChangedAt, true)} behind source`
```
would give you a relative time to the source like "3 days behind source".